encryptio.py

```diff
@@ -302,7 +282,7 @@ def decrypt_blowfish(data, key, mode):
             print('This mode is not available for the selected algorithm')
             return
 
-    print(plaintext)
+    print("Decryption successfull: " + plaintext)
```

Comment:
this throws a TypeError: can only concatenate str (not "bytes") to str

Reply:
Resolved this problem by encoding plaintext bytes to utf-8.

Reply:
Looks good 👍